addons/mail/static/src/discuss/call/common/thread_model_patch.js

```diff
@@ -4,6 +4,12 @@ import { browser } from "@web/core/browser/browser";
 
 import { patch } from "@web/core/utils/patch";
 
+export const INDICATOR = Object.freeze({
+    DISABLED: "DISABLED",
+    ACTIVE: "ACTIVE",
+    DISCARDED: "DISCARDED",
+});
```

Comment:
I would rename `INDICATOR` to `CALL_PROMOTE_FULLSCREEN`, again to give a better grasp by its name, especially when this is exported thus it's globally available in IDE autocompletion.

And would rename `DISABLED` to `INACTIVE`, so symmetry with `ACTIVE`.